### PR TITLE
8.0 web_translate_dialog: Add author and license key in __openerp__.py

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

File: web_translate_dialog/__openerp__.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
 {"name": "Web Translate Dialog",
  "category": "Hidden",
```
Comment (pedrobaeza) [inline web_translate_dialog/__openerp__.py]: Take the occassion to change this category to a more relevant one

**Comment by lmignon** [comment]
:+1:

**Comment by oihane** [comment]
:+1:
@pedrobaeza you can now merge it

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**